man/man8/zpool.8

```diff
@@ -910,6 +911,11 @@ See
 .Xr spl-module-parameters 5
 for additional details.  The default value is
 .Sy off .
+.It Sy l2cache_persistent Ns = Ns Sy on Ns | Ns Sy off
+Controls whether the L2ARC is rebuilt and thus made persistent across reboots.
```

Comment:
This scentence reads a little weird...
Maybe something like this is more clear:
`Controls whether the L2ARC is rebuilt at reboot and thus made persistent.`